tests/test_rma.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+import ucp
+
+
+@pytest.mark.parametrize("blocking_progress_mode", [True, False])
+def test_fence(blocking_progress_mode):
+    ucp.init(blocking_progress_mode=blocking_progress_mode)
+    # this should always succeed
+    ucp.fence()
+    ucp.reset()
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("blocking_progress_mode", [True, False])
+async def test_flush(blocking_progress_mode):
+    ucp.init(blocking_progress_mode=blocking_progress_mode)
+
+    await ucp.flush()
+    ucp.reset()
+
+
+@pytest.mark.parametrize("blocking_progress_mode", [True, False])
+def test_worker_address(blocking_progress_mode):
+    ucp.init(blocking_progress_mode=blocking_progress_mode)
+
+    addr = ucp.get_worker_address()
+    assert addr is not None
+    ucp.reset()
```

Comment:
A test that communicate `UCXAddress` between two endpoints would be good. It will show that the buffer protocol works and that different processes can read the address.

Reply:
+1 to this, the easiest is probably just to copy https://github.com/rapidsai/ucx-py/blob/5208b4538a86e2d1e22b596a5050943aeae70bcd/tests/test_send_recv.py#L56-L71 and modify it to pass the address.